code/datums/ai_laws/ai_laws.dm

```diff
@@ -2,7 +2,14 @@
 
 ///What lawset type is considered the "default" ones. In most cases, this is just asimov, but a few circumstances change that.
 ///Requires config, which isn't loaded at the time global vars init, so setup_round_default_laws is called by the first request to use this global.
-GLOBAL_VAR(round_default_lawset)
+///Do NOT get this directly! It's unsafe! Instead, use `get_round_default_lawset`!
+GLOBAL_VAR(_round_default_lawset)
```

Comment:
Why not just use a static var inside `get_round_default_lawset`?

Reply:
Shore!